src/test/regress/expected/citus_local_tables.out

```diff
@@ -635,7 +635,7 @@ FROM pg_dist_partition WHERE logicalrelid = 'citus_local_table_4'::regclass;
 SELECT column_name_to_column('citus_local_table_4', 'a');
                                                   column_name_to_column
 ---------------------------------------------------------------------
```

Comment:
do we have any tests with column_to_column_name in PG 16? If possible, would be nice to add to upgrade tests to show that everything is fine for all version upgrades

Reply:
Adding upgrade tests for this makes sense, I will do that.